project/urls.py

```diff
@@ -23,7 +23,9 @@
     path("", IndexView.as_view(), name="index"),
     path("members/", include("members.urls")),
     path("calendar/", include("events.urls")),
-    path("news/", include("news_app.urls")),
-    path("chats/", include("chats.urls")),
-    path("admin/", admin.site.urls),
+    path('news/', include("news_app.urls")),
```

Comment:
```suggestion
    path('news/', include("news_app.urls")),
```
這裡這幾行要用 `''` 還是 `""`？